core/src/main/resources/jenkins/model/Messages.properties

```diff
@@ -41,7 +41,7 @@ Hudson.NotUsesUTF8ToDecodeURL=\
   Your container doesn\u2019t use UTF-8 to decode URLs. If you use non-ASCII characters as a job name etc, \
   this will cause problems. \
   See <a href="https://jenkins.io/redirect/troubleshooting/utf8-url-decoding">Tomcat i18n</a> for more details.
-Hudson.NodeDescription=the master Jenkins node
+Hudson.NodeDescription=the Jenkins controller''s built-in node
```

Comment:
(for other reviewers)

![Screenshot_2021-05-18_101012_001](https://user-images.githubusercontent.com/2662497/118615731-4f2f4080-b7c1-11eb-8de6-f5d6500bfb70.png)

it's working fine ^^